src/main.ts

```diff
@@ -26,10 +30,15 @@ async function run(): Promise<void> {
     if (games.length === 0) {
       throw new Error('No games found!');
     }
-
+    const stats: Stats = await getStats(CHESS_USERNAME);
     console.log(games.length + ' games found!');
 
-    const gamesString = formatTable(games, CHESS_USERNAME, SHOW_DATE, SHOW_FEN);
+    const reportString = `${formatTable(
+      games,
+      CHESS_USERNAME,
+      SHOW_DATE,
+      SHOW_FEN
+    )}\n${SHOW_STATS && formatStatsTable(stats)}`;
```

Comment:
Can you please test what happens if SHOW_STATS is not set?
I think this will print the string `false` in the final output.

Reply:
yes it shows false.